packages/zipkin/src/internalRepresentations.js

```diff
@@ -1,8 +1,9 @@
 const thriftTypes = require('./gen-nodejs/zipkinCore_types');
 const {now, hrtime} = require('./time');
 const {Some, None} = require('./option');
+const {getAddressByName} = require('./InetAddress');
 
-function Endpoint({serviceName, host, port}) {
+function Endpoint({serviceName, host, port} = {}) {
```

Comment:
This was added to prevent the need of doing `new Endpoint({})` below, but I'm not happy with both variants. Will revert it, for now.